src/main/java/core/training/TrainingPreviewPlayers.java

```diff
@@ -112,7 +113,7 @@ public int getNextWeekTraining() {
                 nextWeekWeek++;
             }
             var hattrickDate = new HattrickDate(nextWeekSaison, nextWeekWeek);
-            nextWeekTraining = DBManager.instance().getFuturTraining(hattrickDate.toInstant());
+            nextWeekTraining = DBManager.instance().getFuturTraining(Timestamp.from(hattrickDate.toInstant()));
```

Comment:
,does the one from training manager not good?

Reply:
sorry - but i didn't got it that the training manager would also load future trainings from futureTrainingTable

Reply:
no it only load upcoming training for subskill calculation.